quacc/calculators/qchem.py

```diff
@@ -0,0 +1,177 @@
+"""
+A Q-Chem calculator built on Pymatgen and Custodian functionality
+"""
+from __future__ import annotations
+
+import inspect
+import os
+import struct
+
+from ase import Atoms, units
+from ase.calculators.calculator import FileIOCalculator
+from monty.io import zopen
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.io.qchem.outputs import QCOutput
+from pymatgen.io.qchem.sets import ForceSet
+
+from quacc.custodian import qchem as custodian_qchem
+
+
+class QChem(FileIOCalculator):
+    """
+
+    Parameters
+    ----------
+    input_atoms
+        The Atoms object to be used for the calculation.
+    cores
+        Number of cores to use for the Q-Chem calculation.
+    charge
+        The total charge of the molecular system.
+        Effectively defaults to zero.
+    spin_multiplicity
+        The spin multiplicity of the molecular system.
+        Effectively defaults to the lowest spin state given the molecular structure and charge.
+    qchem_input_params
+        Dictionary of Q-Chem input parameters to be passed to pymatgen's ForceSet.
+    use_custodian
+        Whether to use Custodian to run Q-Chem.
+        Default is True in settings.
+    **kwargs
+        Additional arguments to be passed to the Q-Chem calculator. Takes all valid
+        ASE calculator arguments, in addition to those custom to Quacc.
```

Comment:
Not sure this line is relevant here since you aren't inheriting from the ASE Q-Chem calculator. Probably worth clarifying the code comment a bit, e.g. referring to the corresponding Pymatgen section.

Reply:
Changed to "Additional arguments to be passed to the FileIOCalculator."